app/controllers/api/v1/screenshots_controller.rb

```diff
@@ -8,7 +8,8 @@ class ScreenshotsController < ApplicationController
 
       def index
         @screenshots = Screenshot.where(product_version_id: @version, locale_id: @locale)
```

Comment:
I am not sure if we are going to use the `screenshots` variable else where, is it possible to use it as a local variable rather than an instance variable?

Reply:
Also, few more improvements as a part of this PR could be:
a) Line 32, uses the `find` method, what if the value is `nil`?
b) Line 11, what if the @screenshots value is nil?

Reply:
Thanks for the review. Regarding "@screenshots"  variable for now we won't be using it else where but what if we require it to use in future. 
For line 11 and 32(find method), we can have Rescue method (rescue ActiveRecord::RecordNotFound)(similar to try..catch block) or else we can have global level error handling. Please refer [this](https://medium.com/rails-ember-beyond/error-handling-in-rails-the-modular-way-9afcddd2fe1b) for reference for this 2 ways.

Reply:
> Thanks for the review. Regarding "@Screenshots" variable for now we won't be using it else where but what if we require it to use in future.

I don't see a future use, if you do so please give an instance, else we can get rid of the instance variables :)

> For line 11 and 32(find method), we can have Rescue method (rescue ActiveRecord::RecordNotFound)(similar to try..catch block) or else we can have global level error handling. Please refer [this](https://medium.com/rails-ember-beyond/error-handling-in-rails-the-modular-way-9afcddd2fe1b) for reference for this 2 ways.

We never use global level error handling like `rescue Exception => e`. Check the type of error that you receive while passing a nil value and use that to rescue. Check this out for further reference: https://guides.rubyonrails.org/action_controller_overview.html#rescue

Reply:
Thanks for providing the reference :) 
Can you please review the code. I did some changes for error handling.

Reply:
The error handling is fine, could you think of a further enhancement with the instance variables?

Reply:
yes. As discussed above, we don't need instance variable for screenshots. Changed it to local variable.